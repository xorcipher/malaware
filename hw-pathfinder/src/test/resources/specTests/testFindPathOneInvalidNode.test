# Find a path with Dijkstra's algorithm when
# one of the nodes searched for is invalid.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 5.0

FindPath simpleGraph A C