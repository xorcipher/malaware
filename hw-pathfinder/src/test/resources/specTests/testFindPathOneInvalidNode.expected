# Find a path with Dijkstra's algorithm when
# one of the nodes searched for is invalid.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 5.000 from A to B in simpleGraph

unknown node C